sgx/demo/app/kmyth_sgx_retrieve_key_demo.c

```diff
@@ -51,7 +51,8 @@
 /* These parameters are hard-coded for now. */
 #define SERVER_IP "localhost"
 #define SERVER_PORT 7000
-#define KEY_ID "fake_key_id"
```

Comment:
```fake_key_id``` was a good, descriptive string for the testing. Is there a compelling reason for moving to "7"?

Reply:
In PyKMIP, the unique key identifiers appear to be automatically-assigned database IDs, so they're decimal strings starting at `1` with the first key registered.

My PyKMIP setup instructions register 10 keys, so any value 1-10 would work for this test.

We could change the demo from get-by-id to get-by-name, but it would be a bit more complicated and probably require two separate KMIP requests.